mappings/net/minecraft/advancement/Advancement.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS q net/minecraft/advancement/SimpleAdvancement
+CLASS q net/minecraft/advancement/Advancement
```

Comment:
Uh, when did we decide this

Reply:
There are no other advancement classes or interfaces. This class is named wrongly, every advancement is a `SimpleAdvancement`